intranet/static/js/eighth/distribute_group.js

```diff
@@ -37,49 +38,49 @@ $(function() {
     });
 
     distribute = function() {
-        var acts = [];
-        var act_names = []
+        var acts = [],
+            act_names = [];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'act_names' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2746690618/issues/source?bid=3212528&fileBranchId=3355681#l42)